public/js/views/management/subscriptions.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component, PropTypes } from 'react';
+import SubscriptionList from 'components/subscription-list';
+
+import { gettext } from 'utils';
+
+
+export default class Subscriptions extends Component {
+
+  static propTypes = {
+    getUserSubscriptions: PropTypes.func.isRequired,
+    userSubscriptions: PropTypes.array.isRequired,
+  };
+
+  componentDidMount() {
+    if (this.props.userSubscriptions === null) {
+      this.props.getUserSubscriptions();
+    }
```

Comment:
oh yeah, this is a cool way to do it. The only downside I can think of is that all actions that alter subscriptions (cancel, add, etc) will need to nullify `user.subscriptions` but I think that will probably happen naturally.